package.json

```diff
@@ -2,9 +2,8 @@
   "name": "tag-release",
   "version": "5.19.0",
   "description": "Automatically Tag and Release Repositories",
-  "main": "index.js",
+  "main": "./src/index.js",
```

Comment:
I wonder if we should keep the root index and point it back there?

Reply:
we can talk about this, not sure what you mean. =)